src/loadgenerator/locustfile.py

```diff
@@ -47,22 +50,30 @@ def addToCart(l):
     l.client.get("/product/" + product)
     l.client.post("/cart", {
         'product_id': product,
-        'quantity': random.choice([1,2,3,4,5,10])})
+        'quantity': random.randint(1,100)})
+    
+def empty_cart(l):
```

Comment:
**Thought**:
Other method names use camelCase, but `empty_cart` is good.
We should use snake_case for Python function names, based on this [StackOverflow post](https://stackoverflow.com/a/8908798).
No action needed (since this is out-of-scope for your change and you've gotten it right).

Reply:
I've opened https://github.com/GoogleCloudPlatform/microservices-demo/issues/2397. No action needed from this pull-request.